pom.xml

```diff
@@ -655,9 +662,9 @@
             <scope>provided</scope> <!-- this dependency is provided by the servlet container -->
         </dependency>
         <dependency>
-            <groupId>javax.annotation</groupId>
-            <artifactId>jsr250-api</artifactId>
-            <version>1.0</version>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.3.1</version>
```

Comment:
Why is this dependency being changed? Is this necessary for the Geotools update or some separate issue?

Reply:
I needed to add it in due course. I should indeed challenge it now